net-proxy/squidguard/files/squidguard-1.5_beta-db5.patch

```diff
@@ -0,0 +1,12 @@
+diff -ruN squidGuard-1.4-vanilla/src/sgDb.c squidGuard-1.4/src/sgDb.c
+--- squidGuard-1.4-vanilla/src/sgDb.c	2008-07-15 04:29:41.000000000 +1000
++++ squidGuard-1.4/src/sgDb.c	2013-01-21 12:47:41.049325756 +1100
```

Comment:
This doesn't look like fixing a test failure. This looks like fixing a broken program, so adjust the commit message appropriately.